client/src/components/DataFiles/DataFilesModals/DataFilesShowPathModal.jsx

```diff
@@ -11,6 +11,25 @@ const DataFilesShowPathModal = React.memo(() => {
     shallowEqual
   );
 
+  const modalParams = useSelector(
+    (state) => state.files.params.modal,
+    shallowEqual
+  );
+
+  const modalApi = useSelector(
+    (state) => state.files.params.modal.api,
+    shallowEqual
+  );
```

Comment:
This could be simplified a bit by not calling the selector twice:
```suggestion
const { api: modalApi } = modalParams;
```

Reply:
Makes sense! Feel free to commit that into this branch if you want.